akka-http-core/src/main/scala/akka/http/scaladsl/model/Multipart.scala

```diff
@@ -467,6 +470,14 @@ object Multipart {
           FastFuture.successful(this)
         override def productPrefix = "FormData.BodyPart.Strict"
       }
+
+      /** Java API */
+      object StrictBuilder {
+        def apply(_name: String, _entity: HttpEntity.Strict,
```

Comment:
Part of the problem is that there is no `Multipart.scala` in `javadsl`. Only `Multipart.java` but this just contains the interface. If you feel that this would be better the helper functions can be extracted to separate `Multipart.scala` created in `javadsl`. That probably would make sense.